proto/profile/events.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+package tendermint.spn.profile;
+
+import "gogoproto/gogo.proto";
+import "profile/coordinator.proto";
+import "profile/validator.proto";
+
+
+option go_package = "github.com/tendermint/spn/x/profile/types";
+
+message EventCoordinatorCreated {
```

Comment:
I think it makes sense for these "custom" events to be more descriptive about the state transition (e.g. a `beforeValidator` and `afterValidator` in `EventValidatorUpdated`) instead of a single entry.

Reply:
I may have overengineerd this in my head. If you are tracing on-chain events you are pretty much guaranteed to have the previous data, so you only need to know the new one in the event.